cub/examples/device/example_device_select_if.cu

```diff
@@ -43,6 +43,8 @@
 #include <cub/device/device_select.cuh>
 #include <cub/util_allocator.cuh>
 
+#include <limits>
+
```

Comment:
Should we move towards ::cuda::std::numeric_limits?

Reply:
I thought so too, but in the example it wasn't necessary, and maybe `<limits>` is more familiar. I really don't mind.

Reply:
Fine for me